### PR TITLE
Notify the user if there is no memory statistic data.

### PR DESCRIPTION
**Pull request by knightburton**
IoT.jsCode-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

**Comment by knightburton** [comment]
@daeyeon Do you mean the text inside the logger calls? The ESlint will warn you if you do not use interpolation inside a template literal (do not forget that, if you indent the string inside a template literal that will be visible in the output) and if we want to be within 120 character per line we have to split the texts, so the conventional way is better in these cases. I would like to use only template literals, but I think this is better solutuion.

**Comment by daeyeon** [comment]
@knightburton Yes, I thought that you'd thought like that, and I also agree that. Calling `this._chart.addNewDataPoints` is also related.

File: src/app/client-connection.js [hunk 1]
```diff
@@ -243,29 +243,39 @@ function onmessage(event) {
...
+              '#' +
+              this._session.getBreakpointInfoToChart().split(':')[1].split(' ')[0] + ': ' +
+              new Date().toISOString().slice(14, 21)
+            );
+          } else {
+            this._chart.addNewDataPoints(messagedata, breakpointLineToChart);
+          }
         }
+      } else {
+        // Notify the user about that, propbably the jerry was builded without the memory statistic swicth.
```
Comment (rtakacs) [inline src/app/client-connection.js]: `builded` -> `built`

File: src/app/client-connection.js [hunk 2]
```diff
@@ -243,29 +243,39 @@ function onmessage(event) {
...
+        this._logger.error(
+          'There is no available memory statistic data. ' +
+          'If you want to use the memory usage panel check the engine build command first.',
```
Comment (rtakacs) [inline src/app/client-connection.js]: There is no memory statistics data available.

File: src/app/client-connection.js [hunk 3]
```diff
@@ -243,29 +243,39 @@ function onmessage(event) {
...
-        } else {
-          this._chart.addNewDataPoints(messagedata, breakpointLineToChart);
+        if (this._session.getBreakpointInfoToChart() && this._chart.isChartActive()) {
+          let breakpointLineToChart = 'ln: ' + this._session.getBreakpointInfoToChart().split(':')[1].split(' ')[0];
+
+          if (this._debuggerObj.getEngineMode() === ENGINE_MODE.BREAKPOINT) {
+            this._chart.addNewDataPoints(
+              messagedata,
+              '#' +
+              this._session.getBreakpointInfoToChart().split(':')[1].split(' ')[0] + ': ' +
```
Comment (rtakacs) [inline src/app/client-connection.js]: What about to store the value of `this._session.getBreakpointInfoToChart().split(':')[1].split(' ')[0]` into a variable. In this case you could avoid a code duplication in this function.

**Comment by knightburton** [comment]
@rtakacs I've updated the PR based on your requests.

**Comment by rtakacs** [comment]
LGTM

**Pull request merged**